packages/block-library/src/paragraph/style.scss

```diff
@@ -41,4 +41,5 @@ p.has-background {
 
 p.has-text-color a {
```

Comment:
The "2013" explicitly disables the underline by using adding a "text-decoration: none;" to anchors. This rule is basically overwriting the theme rule. I think this probably should probably be addressed at the theme level. Any thoughts on this @kjellr, @mapk?

Reply:
> This rule is basically overwriting the theme rule.

Just to clarify, this change is intended to override those themes which explicitly disable underline, including Twenty Thirteen; all of the ones which I tested disable underline, but use a colour to distinguish the link.  

When inside a paragraph block with colour, Gutenberg overrides that colour, but doesn't do anything else to distinguish the link from ordinary text. This would change that. 

> I think this probably should probably be addressed at the theme level.

That would require all 6500 or so themes which do the above to insert the same CSS though, which doesn't seem ideal when it's hard to imagine a negative scenario to this change! :)